manifests/init.pp

```diff
@@ -364,6 +368,7 @@
   String                         $service_group                   = 'root',
   # gitlab specific
   String                         $rake_exec                       = '/usr/bin/gitlab-rake',
+  Optional[Hash]                 $alertmanager                    = undef,
```

Comment:
Can this default to an empty hash please instead of undef?

From our docs at https://voxpupuli.org/docs/#reviewing-a-module-pr : 
> Are there new params with datatype Hash or Array? If possible, they should default to empty Hash/Array instead of undef. You can also enforce the datastructure like Array[String]

Can you please enforce the data structure?

Reply:
`undef` is correct for this module. And empty hash would change the behavior of the installation. I believe you've been given this same feedback in several other PR's.

Reply:
@bastelfreak sorry.. there was undoubtedly a better way to say that. I didn't _mean_ to be such a jerk.